sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java

```diff
@@ -370,7 +370,7 @@ public void registerCoder(Class<?> clazz, CoderFactory coderFactory) {
    *
    * <p>See {@link #getFallbackCoderProvider}.
    */
-  public void setFallbackCoderProvider(CoderProvider coderProvider) {
+  void setFallbackCoderProvider(CoderProvider coderProvider) {
```

Comment:
This one is definitely public. It enables users to set various universal fallbacks beyond those that we configure when we ship.